apps/test/netsim/NetSimRouterNode.js

```diff
@@ -684,6 +698,127 @@ describe("NetSimRouterNode", function () {
       });
     });
 
+    describe("Router memory limits", function () {
+      var sendMessageOfSize = function (messageSizeBits) {
+        var payload = encoder.concatenateBinary({
+          toAddress: intToBinary(remoteA.address, 4),
+          fromAddress: intToBinary(localClient.address, 4)
+        }, '0'.repeat(messageSizeBits - 8));
```

Comment:
I did a while back; it's just so handy for writing string-padding operations, and for generating test data.